packages/legend-application-query/src/stores/QueryEditorStore.ts

```diff
@@ -211,6 +216,9 @@ export abstract class QueryEditorStore {
   initState = ActionState.create();
   queryBuilderState?: QueryBuilderState | undefined;
   exportState?: QueryExportState | undefined;
+  isQueryLoaderOpen = false;
+  loadQueriesState = ActionState.create();
+  queries: LightQuery[] = [];
 
   constructor(
     applicationStore: LegendQueryApplicationStore,
```

Comment:
Hm, let's organize these into `QueryLoaderState` like what we did in `QueryExportState`